### DISCUSSION TITLE
MPPsolar MPI10

### DISCUSSION THREAD

**Discussion by Desacrel** [softwarecrash/Solar2MQTT-ESP8266#101] [General]
Hello, I have been following this project for a long time now and wanted to ask to what extent this project can be used for me.

I have an MPI10k from MPPSolar. (with 3 phases)

The smaller models are already listed, unfortunately mine is not.
Therefore I would be interested to know whether it is simply not possible or whether no one has asked or has it yet. Maybe I can help somehow.

Hence the question as to whether anyone already has it running.

An rs232 or USB interface would be possible for me. I also have hardware there or would order it ^^

Greetings Desa

**Comment by softwarecrash** [comment]
Have you checked if your device working with the PC Software watchpower or Solarpower over the rs232 Port?

**Comment by Desacrel** [comment]
Yes, but not directly. An included WiFi dongle is used via RS232. and then I only have the direct USB-B port. I tried to access the signal there, but unfortunately it doesn't activate if no device such as a laptop is connected. If I understand correctly, this is due to a pullup resistance.

Unfortunately, I don't know enough about it to remove this. But I'm open to everything ^^

**Reply by softwarecrash** [reply]
connect the esp with a rs2323 board to the rs232 port from the wifi dognle, the usb port is not useable for esp.
